### PR TITLE
Modernize

### PR DESCRIPTION
**Pull request by hannesa2**
Please see commit messages

### CODE REVIEW

File: owncloudComLibrary/build.gradle [hunk 1]
```diff
@@ -25,20 +25,13 @@ android {
         minSdkVersion 21
         targetSdkVersion 29
 
-        versionCode = 10001200
-        versionName = "1.0.12"
```
Comment (hannesa2) [inline owncloudComLibrary/build.gradle:29]: Android Arctic Fox doesn't use this any more

**Comment by abelgardep** [comment]
Thanks for the PR! We will take a look right after the scoped storage is merged. We need to upgrade to api 30 first 👍

**Comment by abelgardep** [comment]
Hi @hannesa2, we won't bump the targetSdkVersion to 31 yet. We will wait a little bit since we need to check if it could break something.

**Comment by hannesa2** [comment]
@abelgardep Ok, I removed api 31 commit

**Comment by jesmrec** [comment]
Some check regarding this, using the app's `master` branch:

- App builds correctly
- App builds correctly in "branding" mode
- Tests pass
- App install correctly

For me, no objections to move this forward.

**Review by abelgardep — APPROVED** [review]

**Pull request merged**